### DISCUSSION TITLE
Visual cue for active tab

### DISCUSSION THREAD

**Discussion by TheNicker** [doublecmd/doublecmd#1351] [Suggestions]
It is hard to distinguish active tab, in the following image 'temp' is the active tab. 
I suggest to adjust the text/background color and/or text style, e,g, bold of the active tab
![image](https://github.com/doublecmd/doublecmd/assets/4753283/0dc6b269-9b82-436c-bcf2-e3d9563d0699)

**Comment by ags1234** [comment]
Good idea. Also, this could be an option.

**Comment by KingRabbid** [comment]
Hi, I upvote this! It's even harder to spot the active tab with Dark Mode theme enabled.

Thanks

**Comment by ags1234** [comment]
I also upvote, I think it would be useful to have the active tab clearly differentiated from the rest.

**Comment by Skif-off** [comment]
Isn't this a question for MS?
DC uses a standard system component.

**Comment by alexx2000** [comment]
Maximum that I can do:

![IMG_20231214_163211_001](https://github.com/doublecmd/doublecmd/assets/13401398/ceedaa1d-38e8-43c3-9d3f-5881380316f3)

DC use standard tab control. Without hacks it cannot be changed.

**Reply by KingRabbid** [reply]
Looks good enough, thanks!

**Comment by alexx2000** [comment]
I added a hack. So other variants:
https://imgur.com/a/OEnGlkw
https://imgur.com/a/eMCjUHF
https://imgur.com/a/hazOXn8

**Comment by TheNicker** [comment]
https://imgur.com/a/OEnGlkw seems like a visually elegant solution👌
Making the active tab bold also looks good, might combine these two

I'm suprised that the LCL doesn't specialize active tab handling.

**Reply by JonnyTech** [reply]
I too second the first option and agree with the above comments.

**Reply by ags1234** [reply]
Me too, in this order:
1. Line above: https://imgur.com/a/OEnGlkw
2. Bold text: https://imgur.com/a/hazOXn8

Maybe make an option in the tab settings?
Also, is there the possibility to choose the line color?
Or maybe the active tab font color?

**Reply by alexx2000** [reply]
I plan to use system `selection` style (pressed toolbar button). So it will look good by default. You can test it using latest snapshot.

![11111111](https://github.com/doublecmd/doublecmd/assets/13401398/e1f59d48-30b9-424b-8e3b-ae96f36a555b)

**Comment by TheNicker** [comment]
Just tested it, I can see the blue line, but it is very thin, hard to notice it on 1440p

**Comment by ags1234** [comment]
In the latest build in Windows the tabs look the same, no line above

**Comment by alexx2000** [comment]
@TheNicker 2 pixels line, I can increase it up to 3 pixels.

@ags1234 it is a Windows 10+ only feature.

**Comment by ags1234** [comment]
@alexx2000 But I am using Windows 11 and don't see a line above tabs. See below:

Double Commander
Version: 1.2.0 alpha
Revision: 11111
Commit: a50a39b4b
Build date: 2023/12/21
Lazarus: 2.2.6.0
FPC: 3.2.2
Platform: x86_64-Win64-win32/win64
OS version: Windows 11 22H2 x86_64

**Comment by alexx2000** [comment]
Maybe are you using dark mode?

**Comment by ags1234** [comment]
@alexx2000 Yes, I was using dark mode. If I turn it off, I see the blue line above the tab, but it is very thin, like @TheNicker said.

**Comment by alexx2000** [comment]
I increased a line height. It is pretty visible with 1440p now.

**Comment by lars18th** [comment]
This improvement it's very useful. Using it in Windows 11. Thank you. :+1:

However, perhaps you could add also an option to set the text in bold. I know that TABs are standard components, but you can control the text type. So, you can print in bold the current tab (or all tabs). This will improve the readability.

**Comment by advice2020** [comment]
--Deleted this Linux based response, I didn't know this was Windows specific

**Comment by lars18th** [comment]
Hi @advice2020 ,

Because DC is multiplatform (and theme dependent) I suggest that you add one (or more) screen captures to your suggestion. It will be then more easy to understand the problem. 😉

**Reply by alexx2000** [reply]
Please don't mix Linux and Windows versions. Under Linux you can choose any theme that you want.

**Reply by lars18th** [reply]
I haven't done so. I'm using Windows, and this PR focuses on Windows. So, any comment? It's acceptable?

**Comment by advice2020** [comment]
Sorry about that, I deleted my comment above, I didn't realize this was a Windows specific post.
The only indicator that it was is that the drives shown in the image provdied use letters :)

EDIT:
I made the following "feature request" about this topic for Linux users.
The possible solution I mention could probably be used for all versions of DC
Here is the link if anyone is interested
https://github.com/doublecmd/doublecmd/discussions/1537
